config/application.rb

```diff
@@ -16,6 +16,7 @@ class Application < Rails::Application
 
     config.browserify_rails.force = true
     config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] ]'
+    config.i18n.available_locales = %w[en es]
```

Comment:
we might want to make this configurable, or different in the different environments? (ex make it `%w[en]` for production but then make it `%w[en es]` for test and development?

The reason is that if somebody visits the IDP today with a Spanish browser, they'd see a bunch of "NOT TRANSLATED YET" strings